src/main/java/pixelperfect/Game.java

```diff
@@ -33,7 +33,7 @@ public static void main(String[] args) {
    */
   @Override
   public void simpleInitApp() {
-    spaceship = new Spaceship();
+		spaceship = Spaceship.getInstance();
```

Comment:
I don't think this is a good idea, I think you are overreacting with the singleton design pattern. Multiple spaceship might, for whatever reason, co-exist in the game. I suggest we only make only those classes that would otherwise be a Utility class a Singleton.